www/js/survey/multilabel/ModeSelectMenu.jsx

```diff
@@ -0,0 +1,35 @@
+/* A scrollable popover menu for choosing the mode when labeling trips */
+
+import React, { } from "react";
+import { angularize } from "../../angular-react-helper";
+import { func } from "prop-types";
+import { ScrollView } from "react-native";
+import { Menu } from "react-native-paper";
+import { useTranslation } from "react-i18next";
+
+const ModeSelectMenu = ({ anchor, visible, inputParams, chooseFn }) => {
+  const { t } = useTranslation();
```

Comment:
@Abby-Wheelis here is an example of how internationalization will work in the new React components we're writing.

`t` is the translation function and it's received using `useTranslation()`.